src/movies.js

```diff
@@ -50,7 +50,13 @@ function orderByYear(moviesArray) {
 
 // Iteration 6: Alphabetic Order - Order by title and print the first 20 titles
 function orderAlphabetically(moviesArray) {
```

Comment:
There is method called localeCompare that already checks the alphabetical order, so you can use this method instead of the conditions:

```
function orderAlphabetically(moviesArray) {
  const moviesAlphabetically = moviesArray.map((movie) => movie.title);
  moviesAlphabetically.sort((a, b) => {

    return a.localeCompare(b);

  });
  return moviesAlphabetically.splice(0, 20);
}
```